lib/stats.js

```diff
@@ -67,7 +67,7 @@ var stats = {
       var childrens = options.childrens ? stat.cutime + stat.cstime : 0
       var total = 0;
 
-      if(history.utime) {
+      if(typeof history.utime !== 'undefined') {
```

Comment:
Also, history.utime is undefined on the first execution of this function for a given pid.  See the top line of the function: " var history = this.history[pid] ? this.history[pid] : {}" .